ocean_lib/test/test_config_provider.py

```diff
@@ -54,6 +148,16 @@ def test_config_from_text():
     assert config.aquarius_url == "https://another-aqua.url"
 
 
+def test_config__from_text__malfformed_content():
```

Comment:
just a tiny typo here, I assume it should be one f for "malformed".
also I am not partial to the double underscoring, since double underscore has meaning other than separator: https://dbader.org/blog/meaning-of-underscores-in-python and also in some dialects like e.g. Django they are automatically interpreted by internal tools e.g. attr1__attr2 means attr1.attr2 in the database. I would suggest simple underscores, since either way the Setup and Expect are documented. But it's up to you, I'm not *very* adamant about this.

Reply:
thanks, fixed both